executor/ddl.go

```diff
@@ -518,9 +518,7 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 		}
 		schema, ok := dom.InfoSchema().SchemaByID(job.SchemaID)
 		if !ok {
-			return true, infoschema.ErrDatabaseNotExists.GenWithStackByArgs(
-				fmt.Sprintf("(Schema ID %d)", job.SchemaID),
-			)
+			return false, nil
```

Comment:
Why not check schema existence?

Reply:
because of the schema may be dropped, and the job is not the job that we need.